app/presenters/task_list_presenter.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+class TaskListPresenter
+  def initialize(task_list:)
+    @task_list = task_list
+  end
+
+  def id
+    @task_list.id
+  end
+
+  def tasks
+    goal_tasks = @task_list.tasks
+    adhoc_tasks = @task_list.adhoc_tasks
```

Comment:
check if these exist / have been queried and if not preload them here (or throw an error?)